spec/jobs/chat_channels/index_job_spec.rb

```diff
@@ -16,20 +16,17 @@
       end
 
       it "calls index" do
-        described_class.new.perform(chat_channel_id: chat_channel_id)
-        expect(chat_channel).to have_received(:index!)
```

Comment:
We need this expectation when a channel exists.

Reply:
No. In before use
```
allow(chat_channel).to receive(:index!)
```
Without it - fail